Updated docsy theme to latest and fixed all styling issues (#244)
EGI-Federation/documentation: content/en/about/contributing/syntax/_index.md

- [Hugo's shortcodes](https://gohugo.io/content-management/shortcodes/)
- [Docsy theme shortcodes](https://www.docsy.dev/docs/adding-content/shortcodes/)

> ### Adding chapters

Reviewer: I'm not sure how it is for other people, I'm a bit unsure about using the wording chapter here, to me it feels that's really making more sense for books than on the web. Section would sound better to me.

Author: OK, will update. For now I restored the current content.

> ### Adding chapters

Reviewer: I thought you removed/renamed the parts about chapters?
Please replace by section for now.

Reviewer: ```suggestion
### Adding sections
```

Author: I did, but then stashed content-related changes while on call with you.

> To create new chapters in the documentation (visible in left-side navigation

Reviewer: ```suggestion
To create a new section in the documentation (visible in left-side navigation
```

> tree), you have to create new folders under `/content/<language>` and add a file

Reviewer: ```suggestion
tree), you have to create a new folder under `/content/en` and add a file
```

`_index.md` to that folder. The beginning of the markdown file contains a
[front matter](https://gohugo.io/content-management/front-matter/) in YAML, JSON

> or TOML, holding the metadata of the page:

Reviewer: Let's only suggest YALM there, we don't want to mix front header types

Author: OK, will update. For now I restored the current content.

> [front matter](https://gohugo.io/content-management/front-matter/) in YAML, JSON
> or TOML, holding the metadata of the page:

Reviewer: ```suggestion
[front matter](https://gohugo.io/content-management/front-matter/) in YAML
holding the metadata of the page:
```

```yaml
---
title: "Syntax"
description: "Syntax guide for EGI documentation"
type: docs
weight: 30
---
```

The parameter `weight` controls the order of the chapters on the same level in

> the left-side navigation tree. Chapters will appear in descending order of their

Author: It's ascending. Corrected in a subsequent commit.

Will render as:

> {{% pageinfo %}} This is a
> placeholder. {{% /pageinfo %}}

Reviewer: ```suggestion
{{% pageinfo %}} This is a placeholder. {{% /pageinfo %}}
```

Author: This render weird in some places. I discovered that adding a line break will render it aligned in the middle vertically.

{{% alert title="Warning" color="warning" %}} This is a warning. {{% /alert %}}

### Code or shell snippets

The code or instructions should be surrounded with three backticks and include
the file type as a parameter.

The supported languages are dependant on the syntax highlighter, which depends
itself on the mardkown parser.

{{% alert title="Note" color="info" %}} For [Hugo](https://gohugo.io/) the
[goldmark parser](https://github.com/yuin/goldmark) is used and it relies on the
[Prism syntax highlighting](https://prismjs.com/download.html#themes=prism). {{% /alert %}}

The list of languages supported by chroma can be found in their

> [GitHub repository](https://github.com/alecthomas/chroma#supported-languages).

Reviewer: If chroma is no more used this should be replaced/removed.

Author: Will replace it in the new PR. For now I restored the current content under this section.
